cognitedata/reveal
# fix: Fixes a bug where zooming in was incorrect when hovering over overlays.

#### Type of change
![Bug](https://img.shields.io/badge/Type-Bug-red) 


## Description :pencil:
Fixes a bug where `zoomToCursor` and `zoomPastCursor` camera modes were behaving incorrectly when cursor was hovering over html overlays that are children of `domElement` passed to `Cognite3DViewer`. 

## How has this been tested? :mag:

Hovering over loading indicator and trying to zoom in with mouse wheel in Reveal examples package. Previously it was always zooming into top left corner when hovering over any overlay, now works just fine. Also, tested with different sizes of reveal viewer that are positioned differently relative to the browser window.

## Test instructions :information_source:
Go to examples, try to zoom in hovering over loading indicator or AxisViewTool. Also, try doing the same with different `Viewer size` option in GUI. Expected behaviour is that you will zoom to the point under your cursor, not anywhere else. Also, can be tested with preview docs in HtmlOverlayTool example with removed line: `pointer-events: none;` in css style of the label.

## Checklist :ballot_box_with_check:
